src/main/java/com/ait/lienzo/client/core/shape/wires/util/WiresConnectorLabelFactory.java

```diff
@@ -25,7 +26,7 @@ public static WiresConnectorLabel newLabelOnLongestSegment(final String text,
 
     public static class SegmentLabelExecutor
     {
-        private static final double TEXT_WRAP_MAX_WIDTH  = 150d;
+        private static final double TEXT_WRAP_MAX_WIDTH  = 350d;
```

Comment:
The value of `150px` was proposed by UX, IMO it's fine... `350px` seems too much also for me. See UX proposals for text [here](https://issues.jboss.org/browse/JBPM-5566?focusedCommentId=13624717&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13624717).